test/benchmark/benchmark_report.md

```diff
@@ -0,0 +1,925 @@
+# Benchmark Report
+
+Benchmark test settings:
+
+|RPS |Connections|Duration (Seconds)|CPU Limits (m)|Memory Limits (MiB)|
+|-   |-          |-                 |-             |-                  |
+|1000|100        |90                |1000          |2048               |
```

Comment:
does each step (e.g. HTTPRoute = 50 or HTTPRoute = 300) run for 90s ?

Reply:
yes